static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,341 @@
         }
     }
 }
+
+// New gallery stuff
+
+// TODO: put in pasteup
+$multimedia-support-5: #161616;
```

Comment:
Happy to move to `_vars.scss` but would prefer just to put it in paste-up palette, is there documentation on how to do this?